### ISSUE TITLE
[Messenger] Consume command returns Undefined constant "SIGTERM" error if PCNTL module not enabled

### ISSUE THREAD

**Issue by tgalcheva** [symfony/symfony#51836]
### Symfony version(s) affected

6.3.5

### Description

Working on dockerized project with shared prebuilt base image without PCNTL module enabled.

After updating to Symfony latest release 6.3.5 - the messenger:consume command no longer can be executed, it returns  

![image](https://github.com/symfony/symfony/assets/105622872/f227b4be-a938-4e63-a00c-18a3c71727ad)

In the latest release there is a fix for exiting messenger:failed:retry command, which introduces this implementation:
```
public function getSubscribedSignals(): array
    {
        return $this->signals ?? [\SIGTERM, \SIGINT];
    }
```
without checking if the signals are defined. 

With symfony/messenger:6.3.4 there is no such problem.
 
According to PHP documentation:
Process Control support in PHP is not enabled by default. (https://www.php.net/manual/en/pcntl.installation.php)

### How to reproduce

Run a Symfony 6.3.5 project with messenger configuration in an environment with php8.2, without pcntl extension.

Try to run `php bin/console messenger:consume`

**Comment by stloyd** [comment]
Duplicate of: https://github.com/symfony/symfony/issues/51790, fixed in: https://github.com/symfony/symfony/pull/51791

**Comment by tgalcheva** [comment]
> Duplicate of: #51790, fixed in: #51791

Oh, sorry for that, I was checking the opened issues and didn't see this there. In the future I will remember to check in the Closed ones too.
